app/controllers/expenses/CheckEmploymentExpensesController.scala

```diff
@@ -123,6 +123,11 @@ class CheckEmploymentExpensesController @Inject()(authorisedAction: AuthorisedAc
                 case Left(result) => Future.successful(result)
                 case Right(result) =>
                   checkEmploymentExpensesService.performSubmitAudits(model, taxYear, prior)
+
+                  if(appConfig.nrsEnabled){
+                    checkEmploymentExpensesService.performSubmitNrsPayload(model, prior)
```

Comment:
noticed that this should be done in the benefits CYA submit method

Reply:
but maybe might be easier if Ina adds it in from her PR, would she able to do that pretty easily? would it be only be that line?